great_expectations/cli/batch_request.py

```diff
@@ -48,7 +48,7 @@
 def get_batch_request(
     datasource: BaseDatasource,
     additional_batch_request_args: Optional[Dict[str, Any]] = None,
-) -> Dict[str, Union[str, Dict[str, Any]]]:
+) -> Dict[str, Union[str, int, Dict[str, Any]]]:
```

Comment:
@Kilo59 To help me recall, when do we get an `int` for `additional_batch_request_args`?  Thanks!

Reply:
Because of the `limit` keyword to `BatchRequest`
https://github.com/great-expectations/great_expectations/blob/79c6ed9f1fd944bdc96369fcbfdcd969f0f481bc/great_expectations/core/batch.py#L408

Reply:
Oh yes.  Thanks.